### PR TITLE
Add SR2021 game announcement article

### PR DESCRIPTION
**Pull request by sedders123**
Fixes https://github.com/srobo/tasks/issues/635
Fixes https://github.com/srobo/tasks/issues/636

### CODE REVIEW

File: _posts/2020-11-21-sr2021-kickstart-and-rules-published.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Our game this year, _Radars of the Lost Ark_, challenges teams to score points by claiming territories using radios. Teams must navigate the arena scanning for tower transmissions and sending their own transmissions back to claim territories. Teams are awarded points at the end of the match for the maximum extent of their territory at any one time. Each owned territory at the maximum extent is worth 2 points. Full details, including the prizes available this year, are available in the [rulebook][rules].
+
+This year's teams have already been strategising their approach, and we can't wait to see their progress over the year!
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can:
+
+- watch our [livestream][livestream] where we explain the game and how this year's competition will work.
+- download [the presentation][presentation] in which we explain this year’s game and how the competition is running this year.
+- download [the microgames][microgames] which are small activities designed to let you become familiar with the Student Robotics kit.
```
Comment (PeterJCLaw) [inline _posts/2020-11-21-sr2021-kickstart-and-rules-published.md:16]: Is "kit" here the right term? Perhaps "simulator"?

**Review by PeterJCLaw — COMMENTED** [review]
Thanks for putting this together.
One small comment on the article.

Regarding the microgames -- there's quite a few places where the heading is one one page and much of the content on the next. Could we force some page breaks to limit that? I suspect that just forcing the third one ("Motor Movement") onto a fresh page would cascade through sufficiently.

**Comment by PeterJCLaw** [comment]
> Regarding the microgames -- there's quite a few places where the heading is one one page and much of the content on the next. Could we force some page breaks to limit that? I suspect that just forcing the third one ("Motor Movement") onto a fresh page would cascade through sufficiently.

I've twiddled the formatting a bit in the microgames doc. If you reprint it I think you should get good page alignment now.

File: _posts/2020-11-21-sr2021-kickstart-and-rules-published.md (hunk 1 above)
Comment (sedders123) [reply _posts/2020-11-21-sr2021-kickstart-and-rules-published.md]: 970f3591184fdfa433feef4fcdd7d3ef9afe610a

**Comment by sedders123** [comment]
Thanks for fiddling with the formatting @PeterJCLaw. Looks good now!

**Review by PeterJCLaw — DISMISSED** [review]

**Comment by WillB97** [comment]
This is looking great. There is a slight formatting issue with the microgames formatting on the first page with the random word on the right of the image.
2020-11-21-195033_1070x377_scrot

**Comment by PeterJCLaw** [comment]
I've fixed the microgames formatting issue in the original doc; it'll need re-exporting though.

**Review by WillB97 — APPROVED** [review]

**Review by PeterJCLaw — APPROVED** [review]

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**